fili-core/src/main/java/com/yahoo/bard/webservice/web/responseprocessors/FullResponseProcessor.java

```diff
@@ -39,4 +47,21 @@
  *
  */
 public interface FullResponseProcessor extends ResponseProcessor {
+    @Override
+    default void processResponse(JsonNode json, DruidAggregationQuery<?> query, LoggingContext metadata) {
```

Comment:
An alternative is to tranform JSON node to Object node in `DruidPartialDataResponseProcessor`